CHANGELOG.md

```diff
@@ -5,155 +5,104 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 
-## Unreleased as of Sprint 96 ending 2018-10-08
-
-### Added
-- Add Total Space column to Physical Storages [(#283)](https://github.com/ManageIQ/manageiq-schema/pull/283)
```

Comment:
This doesn't look like it was added to the released PR list, now it's just missing.
Can you fix this in a follow-up @JPrause ?